### PR TITLE
changed manifest properties and db column names

### PR DESCRIPTION
**Pull request by Dragon1320**
changed based on the example on & the changes here https://github.com/microsoft/winget-pkgs/commit/a460e378fc9e1001229626071d5663cf341c1e5b

### CODE REVIEW

File: src/server/ghService/import/importPackageUtil.ts [hunk 1]
```diff
@@ -5,41 +5,61 @@ import fetch from "node-fetch";
...
+
+  return mpaths;
+};
+
+// TODO actually check it works
+const getManifestFolderPaths = async (): Promise<string[]> => {
+  const mPaths = await getMPaths();
+  const manifestFolderList: ManifestFolderList[] = [];
+
+  for (let i = 0; i < mPaths.length; i += 1) {
```
Comment (iTweeno) [inline src/server/ghService/import/importPackageUtil.ts:33]: you can do for (let whateverYouWant of mPaths) here

File: src/database/helpers/package.ts [hunk 2]
```diff
@@ -186,33 +188,34 @@ const rebuildPackage = async (id: string, pkg: IBaseUpdate<IPackage> = {}): Prom
...
       Tags: tags,
-      Description: latestManifest.Description,
+      Description: latestManifest.ShortDescription,
       Homepage: latestManifest.Homepage,
       License: latestManifest.License,
       LicenseUrl: latestManifest.LicenseUrl,
     },
 
-    Featured: false,
+    // Featured: false,
```
[the remark this answers is not in the document]
Comment (rlarkin212) [reply src/database/helpers/package.ts:212]: image

File: src/server/ghService/import/importPackageUtil.ts (hunk 1 above)
Comment (rlarkin212) [reply src/server/ghService/import/importPackageUtil.ts:33]: doing that did cause a problem if I remember correctly, also shooo

**Comment by rlarkin212** [comment]
I haven't ran it locally, the dev env don't exist anymore and I don't really want to be spinning up prod before a backup is taken etc.

**Pull request closed without merging**